src/components/ProjectCards/ProjectCard.css

```diff
@@ -0,0 +1,21 @@
+.project-card {
+  width: 165px;
+  border-radius: 0.438rem;
+}
+
+.project-card img {
+  width: 165px;
+  height: 165px;
+  border-radius: 0.438rem;
+  object-fit: cover;
+}
+
+.project-card img:hover {
+  scale: 0.97;
+  transition: scale 100ms ease;
+}
```

Comment:
also not something we need to sort now but I think this might be a departure from the design we have - maybe if Rich is coming back to review things we should prepare a list of changes we want reviewed and have this on there.

I rather think this would probably be a change in the drop-shadow instead of a scaling animation to be in line with their Y2K-y0lo aesthetic